tests/test_helpers/vault_client.py

```diff
@@ -32,9 +32,14 @@ def get_secrets(connector: str) -> Optional[Dict[str, Any]]:
     """Returns a map of secrets for the given connector."""
     if not _client:
         return
-    secrets = _client.secrets.kv.v2.read_secret_version(
-        mount_point=_environment, path=connector
-    )
-    secrets_map = secrets["data"]["data"]
-    logger.info(f'Loading secrets for {connector}: {", ".join(secrets_map.keys())}')
+
+    secrets_map = {}
+    try:
```

Comment:
This needed to be wrapped in a try since the secrets for this connector are under a different path in Vault (staging vs development) which we only want to call from our local machines instead of in the Github test runner.

Reply:
not sure i follow. i don't really see a problem wrapping this in a `try` but i guess it's not clear to me how we're planning on getting the secrets in the GH test runner.

Reply:
Hashicorp has a Github action that we are leveraging to pull the secrets down in the GH test runner. You can see the details here https://github.com/ethyca/fidesops/pull/869/files#diff-3361e85b2ea9a24297809199d4356091e0d599b3313c73775c747c9ee0d7bfceR37.